megatron/core/models/mamba/mamba_layer_specs.py

```diff
@@ -16,6 +18,13 @@
 from megatron.core.transformer.spec_utils import ModuleSpec
 from megatron.core.transformer.transformer_layer import TransformerLayer, TransformerLayerSubmodules
 
+moe = get_moe_module_spec(
+    use_te=True,
+    num_experts=8,  # Can be anything non None
```

Comment:
Could the comment be something like "Can be any positive integer (must not be None)"?

Reply:
Addressed.